website/versioned_docs/version-5.x/module-types.md

```diff
@@ -0,0 +1,51 @@
+---
+id: module-types
+title: single-spa conceptual module types
+sidebar_label: Conceptual Module Types
+---
+
+# Concept: single-spa modules
+
+Single-spa has [different categories](/docs/microfrontends-concept#types-of-microfrontends) of microfrontends. It is up to you where and how you use each of them. However, the single-spa core team has [recommendations](/docs/recommended-setup/#applications-versus-parcels-versus-utility-modules).
+
+Here is how each single-spa module works conceptually. This information should help you understand our [recommendations](/docs/recommended-setup/#applications-versus-parcels-versus-utility-modules).
+
+| application                       | parcel                               | utility                              |
+| --------------------------------- | ------------------------------------ | ------------------------------------ |
+| has multiple routes               | no routes                            | no route                             |
+| declarative API                   | impertive API                        | not registered/mounted by single-spa |
+| renders UI                        | renders UI                           | no UI                                |
+| single-spa managed lifecylces     | custom managed lifecycles            | no DOM elements                      |
```

Comment:
```suggestion
| single-spa managed lifecycles     | custom managed lifecycles            | no DOM elements                      |
```

One thing - it is not strictly true that utility modules do not deal with dom elements. A styleguide, for example, exposes UI components that create DOM elements. It sounds like we might be talking about lifecycles or "mount points" in this table row, so the languaging for utility modules could be "no lifecycle methods"